src/main/java/com/yugabyte/sample/common/CmdLineOpts.java

```diff
@@ -350,6 +350,9 @@ public void initialize(CommandLine commandLine) throws ClassNotFoundException {
       }
       AppBase.appConfig.dbPassword = commandLine.getOptionValue("password");
     }
+    if (commandLine.hasOption("yugabyte_password")) {
+      AppBase.appConfig.ybPassword = commandLine.getOptionValue("yugabyte_password");
```

Comment:
You can put in additional check here: If username is either `yugabyte` or `postgres` then fail saying `--yugabyte_password` is not required. So users will not misuse this option even accidentally.